demisto_sdk/tests/integration_tests/modeling_rules_integration_test.py

```diff
@@ -142,6 +217,10 @@ def test_the_test_modeling_rule_command_pack_not_on_tenant(
         try:
             with requests_mock.Mocker() as m:
                 with SetFakeXsiamClientEnvironmentVars() as fake_env_vars:
+                    m.get(
+                        f"{fake_env_vars.demisto_base_url}/xsoar/about",
+                        json={"demistoVersion": "8.4.0"},
+                    )
```

Comment:
Since we added an API call to retrieve the demisto version of the tenant at the beginning of the command, I had to add a mocked value to the some of the tests.

Reply:
consider adding it to a setup fixture instead, wouldn't be easier?

Reply:
I did that, but then that caused me to do some small refactoring, please take a look at it.